language_detector/main.py

```diff
@@ -1,9 +1,29 @@
+# -*- coding: utf-8 -*-
+
 """This is the entry point of the program."""
 
 from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    """Returns the detected lang of given text."""
+    numOfTimesLanguageOccurs = {}
+
+    for word in text.split():
```

Comment:
text.lower().split()  could increase your count rate, since capitalized words my not be captured.